src/scss/06_components/navigation/_sidebar-menu.scss

```diff
@@ -179,3 +194,136 @@ $-outline-width: 3px;
     }
   }
 }
+
+.sidebar-navigation + .sidebar-navigation {
+  @include breakpoint($breakpoint-l) {
+    margin-top: $spacing-quadruple;
+  }
+}
+
+// Section navigation special-case
+.sidebar-navigation--section-navigation {
+  @include breakpoint(max-width $breakpoint-l) {
+    background-color: $color-silver-light;
+    display: block;
+  }
+  .menu__toggle-button {
+    @include breakpoint(max-width $breakpoint-l) {
+      margin-right: -10px; // Align the button with the first menu toggle button.
+    }
+  }
+
+  .menu--level-0 {
+    @include breakpoint(max-width $breakpoint-l) {
+      padding-left: $spacing;
+      padding-right: $spacing;
+    }
+  }
+
+  .menu--level-0 > .menu__item:last-child {
+    @include breakpoint(max-width $breakpoint-l) {
+      border-bottom: 0;
+    }
+  }
+
+  .menu__item--item-below > .menu__link-wrapper {
+    @include breakpoint(max-width $breakpoint-l) {
+      padding-right: 0;
+    }
+  }
+}
+
+.section-navigation__title {
+  @include breakpoint(max-width $breakpoint-l) {
+    @include font('h5');
+    background-color: $color-silver-light;
+    border-bottom: 0 none;
+    margin-top: 0;
+    padding: 0;
+    width: calc(100% - 54px);
+  }
+
+  a {
+    @include breakpoint(max-width $breakpoint-l) {
+      margin-bottom: 0;
+      margin-top: 0;
+      padding-left: $spacing;
+      padding-right: $spacing;
+    }
+
+    &:before {
+      @include breakpoint(max-width $breakpoint-l) {
+        display: none;
+      }
+    }
+  }
+}
+
+.section-navigation__header {
+  @include breakpoint(max-width $breakpoint-l) {
+    @include font('body');
+    display: flex;
+  }
+
+  .section-navigation__toggle-wrapper {
+    @include breakpoint(max-width $breakpoint-l) {
+      align-items: center;
+      background-color: $color-silver;
+      display: flex;
+    }
+  }
+
+  .menu__toggle-button {
+    display: none;
+
+    @include breakpoint(max-width $breakpoint-l) {
+      background-color: $color-silver;
+      height: 54px;
```

Comment:
I'd recommend using variables for these magic numbers to explain what they are and what similar numbers are related with each other.

Reply:
Moved the values to variable.